test/backend/test_model.py

```diff
@@ -22,6 +23,66 @@ def _get_rnd(self, shape, low=-1.0, high=1.0):
+    seq_at_node = helper.make_node('SequenceAt', ['S','at'], ['Y'])
+    out_value_info = helper.make_tensor_value_info('Y',onnx.TensorProto.FLOAT,[None])
+    a_value_info =  helper.make_tensor_value_info('a',onnx.TensorProto.FLOAT,[2, 1, 2])
+    b_value_info =  helper.make_tensor_value_info('b',onnx.TensorProto.FLOAT,[1, 1, 2])
+    c_value_info =  helper.make_tensor_value_info('c',onnx.TensorProto.FLOAT,[3, 1, 2])
+    at_value_info =  helper.make_tensor_value_info('at',onnx.TensorProto.INT32,[])
+
+    graph = helper.make_graph([seq_construct_node, seq_at_node],
+            name='seq_construct_at_test',
+            inputs=[a_value_info, b_value_info, c_value_info, at_value_info],
+            outputs=[out_value_info])
+    model = helper.make_model(graph, producer_name='backend-test')
+    tf_rep = prepare(model)
+    output = tf_rep.run({'a':a, 'b':b, 'c':c, 'at':0})
+    np.testing.assert_almost_equal(output["Y"], a)
+    output = tf_rep.run({'a':a, 'b':b, 'c':c, 'at':-2})
+    np.testing.assert_almost_equal(output["Y"], b)
+    output = tf_rep.run({'a':a, 'b':b, 'c':c, 'at':2})
+    np.testing.assert_almost_equal(output["Y"], c)
+
+    # test SequenceEmpty, SequenceInsert, and SequenceAt
+    p = np.int32(0)
+    seq_empty_node = helper.make_node('SequenceEmpty', [], ['S'])
+    seq_insert_node1 = helper.make_node('SequenceInsert', ['S','a'], ['S1'])
+    seq_insert_node2 = helper.make_node('SequenceInsert', ['S1','b'], ['S2'])
+    seq_insert_node3 = helper.make_node('SequenceInsert', ['S2','c','p'], ['S3'])
+    seq_at_node = helper.make_node('SequenceAt', ['S3','at'], ['Y'])
+
+    p_value_info = helper.make_tensor_value_info('p',onnx.TensorProto.INT32,[])
+
+    graph = helper.make_graph([seq_empty_node, seq_insert_node1, seq_insert_node2, seq_insert_node3, seq_at_node],
+            name='seq_empty_insert_at_test',
+            inputs=[a_value_info, b_value_info, c_value_info, p_value_info, at_value_info],
+            outputs=[out_value_info])
+    model = helper.make_model(graph, producer_name='backend-test')
+    tf_rep = prepare(model)
+    output = tf_rep.run({'a':a, 'b':b, 'c':c, 'p':p, 'at':0})
+    np.testing.assert_almost_equal(output["Y"], c)
+
+    # test SequenceConstruct, SequenceErase, and SequenceLength
+    seq_construct_node = helper.make_node('SequenceConstruct', ['a', 'b', 'c'], ['S'])
+    seq_erase_node = helper.make_node('SequenceErase', ['S','p'], ['S1'])
+    seq_length_node = helper.make_node('SequenceLength', ['S1'], ['Y'])
+
+    graph = helper.make_graph([seq_construct_node, seq_erase_node, seq_length_node],
+            name='seq_construct_erase_length_test',
+            inputs=[a_value_info, b_value_info, c_value_info, p_value_info],
+            outputs=[out_value_info])
+    model = helper.make_model(graph, producer_name='backend-test')
+    tf_rep = prepare(model)
+    output = tf_rep.run({'a':a, 'b':b, 'c':c, 'p':p})
+    np.testing.assert_almost_equal(output["Y"], 2)
```

Comment:
SequenceConstruct, SequenceErase not been test here.
Should us add only SequenceConstruct and only SequenceErase test case here?

Reply:
SequenceConstruct, SequenceErase are sorted of tested here to have the SequenceLength return correctly. SequenceConstruct creates a new sequence with 3 tensors, SequenceErase removes the tensor at position 0 therefore the length is 2. I think this should be sufficient.

Reply:
SequenceConstruct is also used in test 1, "# test SequenceConstruct and SequenceAt". The test case won't pass if SequenceConstruct is not working properly.

Reply:
A new test case is added for SequenceConstruct and SequenceErase to ensure the correct tensor is erased. Please let me know if additional changes are needed.

Reply:
Thanks~ LGTM~